Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLSchemaBuilder.swift

```diff
@@ -0,0 +1,248 @@
+//
```

Comment:
The spacing in this file is a bit inconsistent. Please only one empty line between functions or methods (unless you have a Mark comment)